iOS/GBViewController.m

```diff
@@ -241,30 +245,82 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
+                                                               mediaType:AVMediaTypeVideo
+                                                               position:AVCaptureDevicePositionUnspecified];
+    _allCaptureDevices = cameraDiscoverySession.devices;
+
+    // Filter only the back cameras into a list used for switching between them
+    NSMutableArray *filteredBackCameras = [NSMutableArray array];
+    for (AVCaptureDevice *device in _allCaptureDevices) {
+        if ([device position] == AVCaptureDevicePositionBack) {
+            [filteredBackCameras addObject:device];
+        }
+    }
+    _backCaptureDevices = filteredBackCameras;
+
     _cameraPositionButton = [[UIButton alloc] initWithFrame:CGRectMake(8,
                                                                        0,
                                                                        32,
                                                                        32)];
+    _changeCameraButton = [[UIButton alloc] initWithFrame:CGRectMake(8,
+                                                                     0,
+                                                                     32,
+                                                                     32)];
     [self didRotateFromInterfaceOrientation:[UIApplication sharedApplication].statusBarOrientation];
     if (@available(iOS 13.0, *)) {
         [_cameraPositionButton  setImage:[UIImage systemImageNamed:@"camera.rotate"
                                                  withConfiguration:[UIImageSymbolConfiguration configurationWithScale:UIImageSymbolScaleLarge]]
                                 forState:UIControlStateNormal];
         _cameraPositionButton.backgroundColor = [UIColor systemBackgroundColor];
+
+        [_changeCameraButton  setImage:[UIImage systemImageNamed:@"camera"
+                                                withConfiguration:[UIImageSymbolConfiguration configurationWithScale:UIImageSymbolScaleLarge]]
+                                forState:UIControlStateNormal];
+        _changeCameraButton.backgroundColor = [UIColor systemBackgroundColor];
     }
     else {
-        UIImage *image = [[UIImage imageNamed:@"CameraRotateTemplate"] imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
-        [_cameraPositionButton  setImage:image
+        UIImage *rotateImage = [[UIImage imageNamed:@"CameraRotateTemplate"] imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
+        [_cameraPositionButton  setImage:rotateImage
                                 forState:UIControlStateNormal];
         _cameraPositionButton.backgroundColor = [UIColor whiteColor];
+
+        UIImage *selectCameraImage = [[UIImage imageNamed:@"CameraTemplate"] imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
+        [_changeCameraButton  setImage:selectCameraImage
+                              forState:UIControlStateNormal];
+        _changeCameraButton.backgroundColor = [UIColor whiteColor];
```

Comment:
Don't bother configuring the change camera button before iOS 13 – the API isn't even supported before that OS version, so it will never be shown anyway. I'd move `_changeCameraButton =` to inside the `@available` if and let it remain `nil` otherwise.